pnpm-lock.yaml

```diff
@@ -782,6 +782,9 @@ importers:
       '@automattic/jetpack-connection':
         specifier: workspace:*
         version: link:../connection
+      '@wordpress/api-fetch':
+        specifier: 6.23.0
+        version: 6.23.0
```

Comment:
@jeherve Is it fine to add this?

Reply:
Yup, no issues here. We can add that new dependency to the package, and when doing so we end up having to modify the workspace's lock file.